### ISSUE TITLE
[FR] Keep Active (Note) Tabs Open

### ISSUE THREAD

**Issue by StefanoRausch** [1C0D/Obsidian-Close-Similar-Tabs#3]
When CST does recognise that the note to open already exists / "lives" in a tab / leaf, the current behaviour is as follows:

- Close the active tab / leaf if it is not pinned.
- Switch to the tab / leaf with the note to open.

Is it possible to implement the following enhancement?:

- Check if the currently active tab / leaf holds a note. If that is the case, then do not close the tab.
- Switch to the tab / leaf with the note to open.

Thank you for the plugin and for considering my feature request.

**Comment by 1C0D** [comment]
No the actual behavior is to let the previous tab opened and not to lose it because it's important to keep the history of made changes. So you can still use ctrl z and tab history navigation. the tab is kept in place the new opened deleted. The fact that's it pinned doesn't interact.
What version have you? Are you using the obsidian installer? it seems like you are talking about the very first version I did even before it was accepted. strange

**Comment by StefanoRausch** [comment]
![image](https://user-images.githubusercontent.com/1460531/228681860-26952b63-5f19-4c54-b0cc-f1338b131732.png)

![image](https://user-images.githubusercontent.com/1460531/228681616-6aff737c-488d-4b2c-aa36-0672fbdfa12f.png)

The behaviour of CST is the same as I did describe above. The new feature "no several empty tabs" is not working. Maybe, because I am on the latest insider version?

**Comment by 1C0D** [comment]
Ok I introduced a bug corrected in next release in version 1.2.0 I checked release history. I fixed it quickly but apparently you got this one. Now you have the last version 1.2.4
Empty tabs is just to have the same behavior on new tabs. or quite the same. If you had a previous empty tab and try to create a new empty one, it's deleting the old one. I added a gif to show behavior to the readme -> new release 1.2.5. Or watch it directly there on the main page
